test/CollapseSpec.js

```diff
@@ -30,21 +30,19 @@ describe('<Collapse>', () => {
   it('Should default to collapsed', () => {
     wrapper = mount(<Component>Panel content</Component>);
 
-    assert.ok(wrapper.find('Collapse').props().in === false);
+    assert.ok(wrapper.find(Collapse).props().in === false);
   });
 
   it('Should have collapse class', () => {
     mount(<Component>Panel content</Component>).assertSingle('.collapse');
   });
 
   describe('from collapsed to expanded', () => {
+    // since scrollHeight is gonna be 0 detached from the DOM
+    sinon.stub(collapseHelpers, '_getScrollDimensionValue').returns('15px');
```

Comment:
**issue:** this needs to go into `beforeEach` if it's to do anything? also, don't we need to unwind this stub/mock somewhere?

Reply:
**suggestion:** instead of mocking this, just update the wrapper itself to set something like `style={{ height: 15 }}` or whatever, such that the logic in question will actually calculate the correct height.

then you can get rid of the helper export